float64[6] left_arm_joint
float64[6] right_arm_joint
float64[2] shengjiang
float64[2] dipan